backend/integrationTesting/integrationSuite.go

```diff
@@ -13,14 +16,21 @@ type IntegrationTestSuite struct {
 }
 
 func (suite *IntegrationTestSuite) SetupSuite() {
-	/** Since the  testSetupAbstraction reuses the container parallel use of this suite would cause conflicts. --> We make sure it's not used in parallel using a mutex */
-	exasol := testSetupAbstraction.Create("./exasol-test-setup-config.json") // file does not exist. --> we use the testcontainer test setup
-	suite.Exasol = &exasol
-	suite.Connection = exasol.CreateConnection()
+	if testing.Short() {
+		suite.T().Skip()
+	}
+	/** Since the testSetupAbstraction reuses the container parallel use of this suite would cause conflicts. --> We make sure it's not used in parallel using a mutex */
```

Comment:
```suggestion
	/** TestSetupAbstraction reuses the container, so parallel use of this suite would cause conflicts. We make sure it's not used in parallel using a mutex. */
```